src/index.ts

```diff
@@ -92,6 +100,36 @@ export function euclid(numberOfTrue: number, numberOfFalse: number): boolean[] {
   return bjorklund(subsequences).reduce((a, b) => a.concat(b), []);
 }
 
+// This algorithm, a variant of the Bresenham line algorithm, returns the "brightest mode" of
+// the "scale" where `first` is treated as larger than `second`.
+// It's based on following the closest approximation of the line y = b/a*x that is strictly below the line.
+function bresenham(a: number, b: number, first: any, second: any): any[] {
```

Comment:
I feel this is overly generic. Just push `true` in place of `first` and `false` in place of `second`. Remove these parameters.